Makefile

```diff
@@ -8,7 +8,7 @@ EXEC_NAME := $(OUTPUT_DIR)/protoc-gen-ext
 SOURCES := $(shell find . -name "*.go" | grep -v test.go)
 VERSION ?= $(shell git describe --tags)
 
-GO_BUILD_FLAGS := GO111MODULE=on CGO_ENABLED=0 GOARCH=amd64
+GO_BUILD_FLAGS := GO111MODULE=on CGO_ENABLED=0 $(shell go env | grep GOARCH)
```

Comment:
Is this change equivalent to using the default GOARCH?

Reply:
Good question, I believe it is. So we could prob remove it altogether then correct?

Reply:
handled this in latest commit.